bundles/org.eclipse.text/src/org/eclipse/text/html/CoreRGB.java

```diff
@@ -0,0 +1,231 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.text.html;
+
+import java.io.*;
+
+/**
+ * Instances of this class are descriptions of colors in
+ * terms of the primary additive color model (red, green and
+ * blue). A color may be described in terms of the relative
+ * intensities of these three primary colors. The brightness
+ * of each color is specified by a value in the range 0 to 255,
+ * where 0 indicates no color (blackness) and 255 indicates
+ * maximum intensity.
+ * <p>
+ * The hashCode() method in this class uses the values of the public
+ * fields to compute the hash value. When storing instances of the
+ * class in hashed collections, do not modify these fields after the
+ * object has been inserted.
+ * </p>
+ * <p>
+ * This class is a clone of org.eclipse.swt.RGB without any UI dependencies
+ * </p>
+ *
+ * @since 3.14
+ */
+
+public final class CoreRGB implements Serializable {
```

Comment:
As it's a new API, let's try to make it as clean as possible from the beginning:
* What about making it a record?
* What about calling it just RGB anyway (and people who need to disambiguize SWT vs text would just use package name)?